src/utils/quote_report_utils.ts

```diff
@@ -142,7 +134,6 @@ export const quoteReportUtils = {
                 sellTokenAddress: logOpts.sellTokenAddress,
                 integratorId: logOpts.integratorId,
                 slippageBips: logOpts.slippage ? logOpts.slippage * BIPS_IN_INT : undefined,
-                zeroExTransactionHash: logOpts.submissionBy === 'metaTxn' ? logOpts.zeroExTransactionHash : undefined,
                 decodedUniqueId: logOpts.submissionBy === 'taker' ? logOpts.decodedUniqueId : undefined,
```

Comment:
Do we want to include `gaslessSwapAmm` here as well since `ExtendedQuoteReportForGaslessSwapAmm` also has `decodedUniqueId` field

Reply:
why yes we do